src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.model;
+
+import java.util.Map;
+
+public class Report {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line after the package declaration.